node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -266,10 +266,15 @@ object Configuration {
     // Validators
     val numValidators =
       get(_.run.numValidators, _.validators.flatMap(_.count), DefaultNumValidators)
-    val bondsFile           = getOpt(_.run.bondsFile, _.validators.flatMap(_.bondsFile))
-    val knownValidators     = getOpt(_.run.knownValidators, _.validators.flatMap(_.known))
-    val validatorPublicKey  = getOpt(_.run.validatorPublicKey, _.validators.flatMap(_.publicKey))
-    val validatorPrivateKey = getOpt(_.run.validatorPrivateKey, _.validators.flatMap(_.privateKey))
+    val bondsFile          = getOpt(_.run.bondsFile, _.validators.flatMap(_.bondsFile))
+    val knownValidators    = getOpt(_.run.knownValidators, _.validators.flatMap(_.known))
+    val validatorPublicKey = getOpt(_.run.validatorPublicKey, _.validators.flatMap(_.publicKey))
+
+    val maybePrivateKeyPath = options.run.validatorPrivateKeyPath.toOption.map(Right(_))
```

Comment:
Why can't the private key path be configured in `rnode.toml`?

Reply:
Hmm, you right. Will do.